backend/main/server/resources/Video.py

```diff
@@ -2,11 +2,28 @@
 from flask import request
 from main.server import db, cache, app
 from main.server.models import Video, VideoSchema
-from flask_jwt import jwt_required
 
 video_schema = VideoSchema()
 videos_schema = VideoSchema(many=True)
 
+def insertVideo(col, data):
+    videoLink = artistLink = username = title = None
+    for i in range(0, len(data)):
```

Comment:
Can this section separate into a generic function? I see that it duplicate with similar sections above without any specific field.

Reply:
I began to do it that way, but I couldn't figure out how to pass each argument to the correct position to input into the db. If you want to take a shot at making it more generic but preserving the mapping, please do, my brain is too small for that right now.

Reply:
Not sure if the syntax is correct but this is what I had in mind. By having a DTO for the input object you can have a generic function that maps the DTO object to the corresponding input data
```
class GalleryDTO:
	artworkLink = None
	artistLink = None
	title = None
	username = None


def getDTOFromColAndData(className, col, data): 
	dto = className()

	for i in range(0, len(data)):
		if hasattr(dto, col[i]):
		dto.col[i] = data[i]
	
	return dto
```

Reply:
I did not know you could do something like that. I'll make it happen. Thanks!

Reply:
I haven't used dataclasses before but I believe they're meant for this reason https://docs.python.org/3/library/dataclasses.html

You can also consider a namedtuple since it's a little less complex to set up https://docs.python.org/3/library/collections.html#collections.namedtuple